docs/reference/mappings.md

```diff
@@ -120,13 +121,14 @@ If absolutely necessary, you can manually set a `precedence` on the mapping (see
 
 Ambassador will respond with a `404 Not Found` to any request for which no mapping exists. If desired, you can define a fallback "catch-all" mapping so all unmatched requests will be sent to an upstream service.
 
-For example, defining a mapping with only a `/` prefix will catch all requests previously unhandled and forward them to an external service:
+For example, defining a mapping with only a `/` prefix will catch all requests previously unhandled and forward them to an external service, bypassing authentication (if configured):
 
 ```yaml
 ---
 apiVersion: ambassador/v0
 kind: Mapping
 name: catch-all
+bypass_auth: true
```

Comment:
In order to maintain the previous semantic "catch-all", should `bypass_auth` be set to false?

Reply:
This was a bit of a shot in the dark. The intent was to show a usage of `bypass_auth`, but this is a questionable use case. I'll probably just remove this (since it defaults to `false` anyway).

Reply:
Removed.